consumer/schemas/inventory.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class Device(BaseModel):
+    id: int
+    serial: str
+    device_id: str
+    is_checkout: bool
+    model: Optional[str]
+    manufacturer: Optional[str]
+    location: Optional[str]
+
+    @classmethod
+    def serialize(cls, device: dict) -> "Device":
```

Comment:
Rather than using a forward reference, I argue you can use 
```python
from __future__ import annotations
```
at the very top of this file. It will then automatically detect future reference and you don't need to provide them as a string anymore. The benefit is that future refs will be integrated in python soon, and then we only need to remove this import statement (though I expect it doesn't do any harm and thus upgrading will go smoothly). See also [here](https://stackoverflow.com/questions/61544854/from-future-import-annotations), I've just tried this and it works nicely.